docs/quickstarts/javascript/react/update-the-application.mdx

```diff
@@ -0,0 +1,128 @@
+                    headers: {
+                        Authorization: `Bearer ${accessToken}`,
+                    },
+                });
+
+                try {
+                    const res = await sensitiveDataResponse.json();
+                    setMessage(res.secretMessage)
+                } catch (e) {
+                    //In case no access is given, the response will return 403 and not return a JSON response
+                    setMessage(sensitiveDataResponse.status)
+                }
+            }
+
+        } catch (e) {
+            console.log(e.message);
+        }
+
+    }, [auth.isLoading, auth.userData?.id_token])
+
+   ...
+
+```
+
+In this portion of the code we create a callback (which will be triggered by a button). The callback will first get our JWT token from the identity provider, using the `auth` object that is obtained from the `useAuth` hook. Then we perform the call to our service sending the authorization token as part of our request's headers (fetch). Finally, we parse the JSON response from the server and set the state of the message variable: if the service returns a `403 Forbidden` error, and the message “No access to sensitive information” will be shown. If no error is returned from the service, the user has access to the protected resource and the message will be shown.
+
+Next we’ll update the `main` section of the app (in the `div` with the className `main`) to include the button that will trigger `accessSensitiveInformation` and an area to show the message.
+
+```javascript
+<div className="main">
+  {isAuthenticated && (
+    <>
+      <div className="top-main">
+        <div className="welcome-message">
+          Welcome {auth.userData?.profile?.email}!
+        </div>
+        <div>
+          {!message && (
+            <button
+              className="primary-button"
+              onClick={() => accessSensitiveInformation()}
+            >
+              Get Sensitive Resource
+            </button>
+          )}
+          <div className="message-container">
+            {message && message !== 403 && (
+              <>
+                <div className="lottie"></div>
+                <div className="message">{message}</div>
+              </>
+            )}
+            {message && message === 403 && (
+              <>
+                <div className="sad-lottie"></div>
+                <div className="message">
+                  No access to sensitive information
+                </div>
+              </>
+            )}
+          </div>
+        </div>
+      </div>
+    </>
+  )}
+</div>
+```
+
+## Test the application
+
+To run both your application and the server in parallel, update the `package.json` in the root folder, and add the following to the `scripts` section:
+
+```
+"scripts": {
+   ...
+   "start:server": "node service/index.js",
+   "start:all": "npm-run-all --parallel start start:server"
```

Comment:
You seem to have a missing step here. You asked the user to yarn install (actually should be yarn add) npm-run-all, but it was in the `service` directory. Instead, you should ask the user to `yarn add npm-run-all` in the main project directory. Then this will work.